File: backend/app/schemas/service.py
from typing import Literal

from app.schemas.base import CamelModel


class ServiceOut(CamelModel):
    id: str
    slug: str
    name: str
    short_description: str
    description: str
    duration_minutes: int
    price: float
    currency: str
    is_price_estimate: bool
    appropriate_for: list[str]
    included: list[str]
    is_online: Literal[True] = True

File: src/lib/types/service.ts
export interface Service {
  id: string;
  slug: string;
  name: string;
  shortDescription: string;
  description: string;
  durationMinutes: number;
  /** Placeholder pricing — confirm real pricing before launch. */
  price: number;
  currency: string;
  isPriceEstimate?: boolean;
  appropriateFor: string[];
  included: string[];
  isOnline: true;
}
